setup.cfg

```diff
@@ -11,48 +9,50 @@ description = Community-developed python astronomy tools
 long_description = file: README.rst
 keywords = astronomy, astrophysics, cosmology, space, science,units, table, wcs, samp, coordinate, fits, modeling, models, fitting, ascii
 classifiers =
-    Intended Audience :: Science/Research
-    License :: OSI Approved :: BSD License
-    Operating System :: OS Independent
-    Programming Language :: C
-    Programming Language :: Cython
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: Implementation :: CPython
-    Topic :: Scientific/Engineering :: Astronomy
-    Topic :: Scientific/Engineering :: Physics
+	Intended Audience :: Science/Research
```

Comment:
Why the change in indentation? 4 spaces seems just right...

Reply:
Fixed